bundles/org.openhab.binding.samsungtv/src/main/java/org/openhab/binding/samsungtv/internal/config/SamsungTvConfiguration.java

```diff
@@ -12,15 +12,19 @@
  */
 package org.openhab.binding.samsungtv.internal.config;
 
+import java.util.Optional;
+
 import org.eclipse.jdt.annotation.NonNullByDefault;
-import org.openhab.binding.samsungtv.internal.handler.SamsungTvHandler;
 
 /**
  * Configuration class for {@link SamsungTvHandler}.
  *
  * @author Pauli Anttila - Initial contribution
  * @author Arjan Mels - Added MAC Address
+ * @author Nick Waterton - added Smartthings, subscription, refactoring
  */
+
+@SuppressWarnings("null")
```

Comment:
Please remove the null suppression and handle it in the code.

Reply:
Removed, but there is nothing to handle, just a warning with no importance.